addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1050,41 +1072,42 @@ options.registry.layout_column = options.Class.extend({
      * last x columns.
      *
      * @private
+     * @param {jQuery} $row - the row in which to update the columns
      * @param {integer} count - positif to add, negative to remove
      */
-    _updateColumnCount: async function (count) {
+    _updateColumnCount: async function ($row, count) {
         if (!count) {
             return;
         }
-
-        this.trigger_up('request_history_undo_record', {$target: this.$target});
+        // TODO: check that this is in fact needed
+        this.trigger_up('request_history_undo_record', {$target: $row});
```

Comment:
Yep, don't understand why that would be needed either, this is part of the option system.

Reply:
Seems like that line was introduced with the layout_column option when it was first created. And works fine without it, I'm removing it then.